packages/data-context/src/data/coreDataShape.ts

```diff
@@ -164,7 +164,12 @@ export interface CoreDataShape {
   cloudProject: CloudDataShape
   eventCollectorSource: EventCollectorSource | null
   didBrowserPreviouslyHaveUnexpectedExit: boolean
-  studio: StudioManagerShape | null
+  studioLifecycleManager?: {
```

Comment:
Would it make sense to put this type in the @types package similar to StudioManagerShape. So that we can have that type enforced in both places it's effectively used?